client/login/wp-login/index.jsx

```diff
@@ -651,10 +653,20 @@ export class Login extends Component {
 				{ isWhiteLogin && (
 					<Step.CenteredColumnLayout
 						columnWidth={ 6 }
+						{ ...( isStudioAppOAuth2Client( oauth2Client ) && { columnWidthHeading: 8 } ) }
```

Comment:
Not sure there's a better way of doing this right now, outside of not using `Step.CenteredColumnLayout` for the log-in screen. (Never mind the conditioning, it will go away at some point next)